collectd/init.sls

```diff
@@ -6,6 +6,9 @@ include:
 collectd:
   pkg.installed:
     - name: {{ collectd_settings.pkg }}
+    {%- if collectd_settings.pkg_version is defined and collectd_settings.pkg_version %}
+    - version: '{{ collectd_settings.pkg_version }}'
+    {%- endif %}
```

Comment:
Actually apt-pinning is the reason, I added these lines.
For example, on squeeze I usually use ``squeeze`` distribution, but in this case, I need ``squeeze-backports``.

Or, I could configure apt-pinning exactly for ``collectd*`` packages. Is that, what you mean?
In this case, the only thing I don't like is that my settings would spread in two places.